### ISSUE TITLE
Native class syntax leads to build time eslint errors

### ISSUE THREAD

**Issue by kalien** [ember-cli/ember-cli#8780]
---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 3.11.0
http_parser: 2.8.0
node: 10.16.0
v8: 6.8.275.32-node.52
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 64
nghttp2: 1.34.0
napi: 4
openssl: 1.1.1b
icu: 64.2
unicode: 12.1
cldr: 35.1
tz: 2019a
os: darwin x64
6.10.3
-bash: yarn: command not found```

Steps to recreate the problem:
1) Create a brand new ember project
2) Generate a component in it
3) Change the component content to something similar to:
export default class MyClassName extends Component {

        myFieldName;

}
In other words make sure the component uses native class.
4) Build the project

Problem: 
The build reports a problem:
"error  Parsing error: Unexpected token ;"
Basically the native class syntax is not recognized.

**Comment by kalien** [comment]
Some comments would be really appreciated here. Is this the current behavior or is it just me (something in my environment)? Is there a workaround? If this is a known issue then some rough idea when a fix would come would be appreciated.
Basically, this stops the project from migrating to native class syntax for Ember components because the build output will be flooded with noise and it will be really hard to filter out the real errors from similar false errors. Thanks

**Comment by jherdman** [comment]
I've found that I had to change my eslint parser. The following steps work for me:

1. `yarn add -D babel-eslint`
2. Add the following to `.eslintrc.js`: `"parser": "babel-eslint"`

Now you should be golden!

**Comment by rwjblue** [comment]
FWIW, this was fixed by https://github.com/ember-cli/ember-cli/pull/8564 and is included in 3.13.0-beta.1.

**Comment by rwjblue** [comment]
Sorry you ran into issues here, going to close as it is resolved in the current beta.
